### ISSUE TITLE
Hassbian base on Debian Buster

### ISSUE THREAD

**Issue by Landrash** [home-assistant/hassbian-scripts#254]
Issue for tracking the Hassbian release based on Debian Buster.

**Comment by Landrash** [comment]
Testing of suites

- [x] AppDaemon - Works but needs a small rework to use the proper wording since it now should use "Long-Lived Access Tokens"
- [ ] Cloud9 - **Broken**. Some rework to use the proper npm/node is required. Error after install 'View not found: /opt/c9sdk/plugins/c9.error/views/error-500.html.ejs'
- [x] custom-component-store
- [x] Duck DNS
- [x] Fail2ban
- [x] Hassbian
- [x] hassbian-script (Needs some work, uses "stretch") #263 
- [x] Home Assistant
- [x] Homebridge - Will be removed before release of buster since it's now deprecated.
- [x] HUE
- [x] LibCEC
- [x] Manager
- [x] MariaDB  #264
- [ ] Monitor
- [x] Mosquitto - Fails installation. No repository available for Buster. Fallback to version distributed by Debian? #262
- [ ] Pi-hole
- [x] PostgreSQL - #265
- [x] MS SQL - #261
- [x]  RaZberry - _Does not have that device, but the script runs fine_
- [x] Samba - User get a question about DHCP, not completly silent anymore.
- [x] Trådfri -  Will be removed before release of buster since it's now deprecated.
- [x] Webterminal

**Comment by Landrash** [comment]
Bugg found with local ca-certificates -> https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=923479

Temporary workaround
```
run sudo c_rehash to fix it.
```

Will be added to the install script as a workaround temporarily.

**Comment by ludeeus** [comment]
Demn you GitHub 😆

**Comment by Landrash** [comment]
My fault. Forgot about linked issues. Should have created a separate issue
for tracking.

**Comment by Landrash** [comment]
Testing of suites. New list for reworked version of hassbian-scripts.

- [x] AppDaemon - Functions but gives error during install. Error in Discord.
- [x] Cloud9
- [x] custom-component-store #284 
- [x] Duck DNS - Fails. Branch working on this 'Landrash-uglyduckling' #277 && #283 
- [x] Fail2ban
- [x] Hassbian
- [x] hassbian-script
- [x] Home Assistant - Upgrades with flags `--dev`/`--beta` does not work. #279 
- [x] HUE
- [x] LibCEC
- [x] Manager #284 
- [x] MariaDB
- [x] Monitor
- [x] Mosquitto - Fails installation. No repository available for Buster. Fallback to version distributed by Debian? #262 
- [x] Pi-hole
- [x] PostgreSQL
- [x] MS SQL
- [x] RaZberry
- [x] Samba 
- [x] Trådfri - OK after #281 
- [x] Webterminal

**Comment by Landrash** [comment]
Package built with #290 
Image being built.
Release package will be uploaded from Auto build.
